project.py

```diff
@@ -15,9 +30,12 @@ def __init__(self):
         self.bindings_factories = [QHexEditBindings]
 
     def update(self, tool):
-        """Allows SIP to find PyQt5 .sip files."""
+        """Allows SIP to find PyQt5/PyQt6 .sip files."""
         super().update(tool)
-        self.sip_include_dirs.append(join(PyQt5.__path__[0], 'bindings'))
+        if PyQt_Version == 'PyQt6':
+            self.sip_include_dirs.append(join(PyQt6.__path__[0], 'bindings'))
+        elif PyQt_Version == 'PyQt5':
+            self.sip_include_dirs.append(join(PyQt5.__path__[0], 'bindings'))
```

Comment:
Should this function raise if using PyQt4?

Reply:
Done, I added `raise ValueError(f"Unsupported PyQt version: {PyQt_Version}.")` in the new code